src/blog-utils/StaticAddonCard/styles.scss

```diff
@@ -79,7 +79,7 @@ $icon-size: 48px;
   & .AddonTitle-author,
   & .AddonTitle-author a,
   & .AddonTitle-author a:link {
-    font-size: $font-size-m;
+    font-size: $font-size-default;
     line-height: 1;
   }
 
```

Comment:
At the bottom of the file, we're using `24px`. We could use `$font-size-heading-xs` instead.

I'll take care of the other one (`20px` in `.AddonTitle`).